contrib/storage-splunk/src/test/resources/logback-test.xml.bak

```diff
@@ -1,59 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Could you please move this file to `/dev-support` folder? this is a good reference template file for the test (than the one in the `/drill-common/src/test/resources/` folder). Thanks.

Reply:
Developers shouldn't introduce additional logback-test files, that one from common module should be used for all tests. If we need to change something for dev purposes, that file can be updated.

Reply:
Files in folder `/dev-support` are not compiled, built, and run-time, and exist to support development, so no test files are added.

Reply:
Yes, but I meant that no need to add it as a reference template since we already have that one that can be modified if required.

Reply:
Perhaps I didn't make that clear. This test file is more representative and can retain only the log output information for the submodules. eg,
`<logger name="org.apache.drill.exec.store.splunk">`